app/scripts/views/base.js

```diff
@@ -389,6 +389,29 @@ define(function (require, exports, module) {
 
     // called after the view is visible.
     afterVisible: function () {
+      // restyle side-by-side links to stack if they are too long
+      // to fit on one line
+      var linkContainer = this.$el.find('.links');
+      if (linkContainer.length > 0) {
+        var containerWidth = linkContainer.width();
+        var halfContainerWidth = containerWidth / 2;
+        var shouldResetLinkSize = false;
+
+        linkContainer.children().each(function (i, item) {
```

Comment:
>  what if we choose to wrap 

Might happen but i don't think we have that in place right now so we don't need to worry.


I feel like the `.children()` code might benefit from explicitly stating what you are trying select (aka stack only the links and not something else.) This request is mostly for code clarity rather than performance.